aws/websites/metrics.pytorch.org/update_dashboards.py

```diff
@@ -95,7 +95,9 @@ def file_path(name: str) -> Path:
             os.remove(file["path"])
             updated = True
 
-    if updated:
-        print("::set-output name=UPDATED_DASHBOARDS::yes")
-    else:
-        print("::set-output name=UPDATED_DASHBOARDS::no")
+    # Open GITHUB_OUTPUT if available through CI, otherwise just write to stdout
+    with open(os.getenv("GITHUB_OUTPUT", "/dev/stdout"), "ab") as fp:
```

Comment:
This is probably the one I'm most worried about, but it's also the one that'll probably get deleted anyways since we plan to deprecate metrics.pytorch.org (see https://github.com/pytorch/pytorch/issues/78951)

Reply:
This is Linux/Macos specific, how about something like the following (also, why do you need append to it as to a binary file?)
```suggestion
    with (open(os.environ["GITHUB_OUTPUT"], "a") if "GITHUB_OUTPUT" in os.environ else sys.stdout) as fp:
```

Reply:
oops only seeing this after I merged 😞